src/analysis/retail/evoker/preservation/modules/talents/EssenceBurst.tsx

```diff
@@ -0,0 +1,133 @@
+import { SpellLink } from 'interface';
+import { t } from '@lingui/macro';
+
+const ESSENCE_COSTS: { [name: string]: number } = {
+  'Emerald Blossom': 3,
+  Echo: 2,
+  Disintegrate: 3,
+};
+
+class EssenceBurst extends Analyzer {
+  totalConsumed: number = 0;
+  totalExpired: number = 0;
+  essenceSaved: number = 0;
+  consumptionCount: { [name: string]: number } = { 'Emerald Blossom': 0, Echo: 0, Disintegrate: 0 };
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS_EVOKER.ESSENCE_BURST_TALENT);
+    this.addEventListener(
+      Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.ESSENCE_BURST_BUFF),
+      this.onBuffRemove,
+    );
+    if (this.selectedCombatant.hasTalent(TALENTS_EVOKER.ESSENCE_ATTUNEMENT_TALENT)) {
+      this.addEventListener(
+        Events.removebuffstack.by(SELECTED_PLAYER).spell(SPELLS.ESSENCE_BURST_BUFF),
+        this.onBuffRemove,
+      );
+    }
+  }
+
+  onBuffRemove(event: RemoveBuffEvent | RemoveBuffStackEvent) {
+    const consumeAbility = getEssenceBurstConsumeAbility(event);
+    if (consumeAbility) {
+      const spellName = consumeAbility.ability.name;
+      this.totalConsumed += 1;
+      this.essenceSaved += ESSENCE_COSTS[spellName];
+      this.consumptionCount[spellName] += 1;
+    } else if (event.type === EventType.RemoveBuff) {
+      console.log(`Buff expired at ${this.owner.formatTimestamp(event.timestamp)}`);
+      this.totalExpired += 1;
+    } else {
+      console.log(`Buff expired at ${this.owner.formatTimestamp(event.timestamp)}`);
+      this.totalExpired += (event as RemoveBuffStackEvent).stack;
+    }
+  }
+
+  renderDonutChart() {
+    const items = [
+      {
+        color: SPELL_COLORS.DISINTEGRATE,
+        label: 'Disintegrate',
+        spellId: SPELLS.DISINTEGRATE.id,
+        value: this.consumptionCount['Disintegrate'],
+        valueTooltip: this.consumptionCount['Disintegrate'],
+      },
+      {
+        color: SPELL_COLORS.EMERALD_BLOSSOM,
+        label: 'Emerald Blossom',
+        spellId: SPELLS.EMERALD_BLOSSOM.id,
+        value: this.consumptionCount['Emerald Blossom'],
+        valueTooltip: this.consumptionCount['Emerald Blossom'],
+      },
+      {
+        color: SPELL_COLORS.ECHO,
+        label: 'Echo',
+        spellId: TALENTS_EVOKER.ECHO_TALENT.id,
+        value: this.consumptionCount['Echo'],
+        valueTooltip: this.consumptionCount['Echo'],
+      },
+    ].filter((item) => {
+      return item.value > 0;
+    });
+    return <DonutChart items={items} />;
+  }
+
+  get suggestionThresholds() {
+    return {
+      actual: this.totalExpired,
+      isGreaterThan: {
+        major: 0,
+      },
+      style: ThresholdStyle.NUMBER,
+    };
+  }
+
+  suggestions(when: When) {
+    when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) =>
+      suggest(
+        <>
+          You are wasting <SpellLink id={TALENTS_EVOKER.ESSENCE_BURST_TALENT.id} /> stacks
```

Comment:
The suggest text should be instructive.
```suggestion
          Try to avoid wasting <SpellLink id={TALENTS_EVOKER.ESSENCE_BURST_TALENT.id} /> stacks.
```